src/FM.LiveSwitch.Mux/Session.cs

```diff
@@ -103,7 +103,7 @@ public bool HasVideo
                 {
                     return client.CompletedConnections.Any(connection =>
                     {
-                        return connection.CompletedRecordings.Any(recording => recording.VideoFileExists);
+                        return connection.CompletedRecordings.Any(recording => recording.VideoFileExists && recording.VideoStartTimestamp.HasValue);
```

Comment:
Does it make sense to do the same change for `HasAudio` above?

Reply:
Actually, I think we can revert this? Now that we are filtering in `MuxVideo` below?

Reply:
@avenema  After reverting this, the muxer fails to mux for 2 of the 4 sets of customer media. The issue is that ffmpeg thinks there is a session video file to mux in the output directory when in reality there is not. No exception is thrown but the app just writes the log statement: `No such file or directory` and exits.

Reply:
@davidgriffithfm got it.

Does it make sense to make the same change to `HasAudio`?

Reply:
Yes, I think it does